tmt/options.py

```diff
@@ -86,4 +99,71 @@ def invoke(self, context):
                 return self._method.invoke(context)
             return super().invoke(context)
 
-    return MethodCommand
+    class AcceptAllMethodCommand(MethodCommand):
```

Comment:
It seems to me that the check for invalid step methods could be implemented at the end of the `_check_method()` by raising an exception if no matching method (including matchig prefix) is found. @FrNecas, what do you think? If so, this could be much simpler approach.

Reply:
Okay, lets leave this for the next release and try to simplify if possible. I dropped the one commit for now and will play around with it a bit more.

Reply:
Good, agreed.